extensions/wikia/PageHeader/templates/PageHeaderController_categories.php

```diff
@@ -0,0 +1,29 @@
+<?php if ( $categories->hasVisibleCategories() ): ?>
+	<div class="page-header__categories">
+		<span class="page-header__categories-in" data-tracking="categories-top-in"><?= $categories->inCategoriesText ?>:</span>
+		<div class="page-header__categories-links">
+			<?= $categories->visibleCategoriesHTML ?>
```

Comment:
We have some links generated in Categories.class and some here in a template (those in dropdown) - could we make it more consistent?